src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -585,6 +585,7 @@ public virtual void Clear()
 
             foreach (NodeModel el in Nodes)
             {
+                el.RaisesModificationEvents = false;
```

Comment:
Sorry. I meant that this line actually fixes something not related to this PR - that evaluation occurs when a workspace is cleared, but still really useful. Is there a YT task related to that issue?